### PR TITLE
(docs) #4768 Add schema examples for migrations

### LINKED ISSUE #4768: queryInterface addColumn using schema
I expected to be able to add a column to a table within a schema by passing the schema through the options parameter, similar to createTable. E.g.

```javascript
queryInterface.addColumn(
    'myTable',
    'test',
    {
        type: Sequelize.STRING,
    },
    {
        schema: 'mySchema'
    }
)
```
With a little digging, you have to pass an object to `tableName` parameter, e.g.
`{tableName: 'myTable', schema: 'mySchema'}`

From the docs i have read it was not obvious how to create a column on a table in a schema, and i expected it to be the same syntax as `createTable`.

Is this expected?
Or should i create a pull request for `addColumn` to have the same syntax of passing a schema as `createTable` ?

### PR DESCRIPTION
**Pull request by oznu**
Added examples to the documentation to show how to use custom schema when executing migrations.

Resolved #4768

### CODE REVIEW

File: docs/docs/migrations.md [hunk 1]
```diff
@@ -203,14 +204,32 @@ queryInterface.addColumn(
...
 ```
 
-### removeColumn(tableName, attributeName, options)
+### removeColumn(tableNameOrOptions, attributeName, options)
 
 This method allows deletion of a specific column of an existing table.
 
 ```js
-queryInterface.removeColumn('Person', 'signature')
+queryInterface.removeColumn('nameOfAnExistingTable', 'nameOfAnExistingAttribute')
```
Comment (sushantdhiman) [inline docs/docs/migrations.md]: Don't change it to `nameOfAnExistingTable` and `nameOfAnExistingAttribute`, Its already clear from method signature. Same goes for schema example below

**Comment by sushantdhiman** [comment]
Ping @oznu

File: docs/docs/migrations.md [hunk 2]
```diff
@@ -203,14 +204,32 @@ queryInterface.addColumn(
...
 This method allows deletion of a specific column of an existing table.
 
 ```js
 queryInterface.removeColumn('Person', 'signature')
+
+// or with an explicit schema:
+
+queryInterface.removeColumn({
+  tableName: 'Person',
+  schema: 'nameOfAnExistingSchema'
```
Comment (sushantdhiman) [inline docs/docs/migrations.md]: Name of scheme need change too

File: docs/docs/migrations.md [hunk 3]
```diff
@@ -203,14 +204,32 @@ queryInterface.addColumn(
     allowNull: false
   }
 )
+
+// or with an explicit schema:
+
+queryInterface.addColumn({
+    tableName: 'nameOfAnExistingTable',
```
Comment (sushantdhiman) [inline docs/docs/migrations.md]: You forgot to change these. Use a real world example here for tableName, schema and attribute

File: docs/docs/migrations.md [hunk 4]
```diff
@@ -119,8 +119,9 @@ queryInterface.createTable(
     }
   },
   {
-    engine: 'MYISAM', // default: 'InnoDB'
-    charset: 'latin1' // default: null
+    engine: 'MYISAM',                     // default: 'InnoDB'
+    charset: 'latin1',                    // default: null
+    schema: 'nameOfTheExistingSchema'     // PostgreSQL only. default: public
```
Comment (sushantdhiman) [inline docs/docs/migrations.md]: change this too

**Comment by oznu** [comment]
Thanks, I updated the examples I added to use real world examples. 

I can update the existing migration examples to use real work examples as well if you want, though that might be for a seperate pull request.

**Pull request merged**